CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+* Firestore Emulator now supports seeding database from export via `--seed_from_export`.
```

Comment:
I don't think we should add this to the changelog yet.  There's no way to actually pass this through the CLI.